src/types/commands.ts

```diff
@@ -1300,6 +1300,7 @@ export const enum CommandResult {
   InvalidTableResize = "InvalidTableResize",
   PivotIdNotFound = "PivotIdNotFound",
   EmptyName = "EmptyName",
+  InvalidDefinition = "InvalidDefinition",
```

Comment:
```suggestion
  InvalidPivotDataset = "InvalidPivotDataset ",
```
nitpick. Might help us debug if we encounter this result in the wilds 🙂